rpi_backlight.py

```diff
@@ -38,97 +39,174 @@ def _get_value(name: str) -> str:
 
 def _set_value(name: str, value: Any) -> None:
     try:
-        with open(os.path.join(PATH, name), "w") as f:
+        with open(os.path.join(path, name), "w") as f:
             f.write(str(value))
     except (OSError, IOError) as err:
         if err.errno == 13:
             _perm_denied()
 
 
-def get_actual_brightness() -> int:
-    """Return the actual display brightness."""
-    return int(_get_value("actual_brightness"))
+def set_brightness_value(value: Any) -> None:
+    if mode == "MODE_TINKERBOARD":
+        _set_value("tinker_mcu_bl", value)
+    elif mode == "MODE_RPI":
+        _set_value("bl_power", value)
 
+def get_brightness_value() -> int:
+    """Return the actual display brightness."""
+    if mode == "MODE_TINKERBOARD":
+        return int(_get_value("tinker_mcu_bl"))
+    elif mode == "MODE_RPI":
+        return int(_get_value("actual_brightness"))
 
 def get_max_brightness() -> int:
     """Return the maximum display brightness."""
-    return int(_get_value("max_brightness"))
-
+    if mode == "MODE_TINKERBOARD":
+        return 255
+    elif mode == "MODE_RPI":
+        return int(_get_value("max_brightness"))
+    
 
 def get_power() -> bool:
-    """Return wether the display is powered on or not."""
+    """Return whether the display is powered on or not."""
     # 0 is on, 1 is off
-    return not int(_get_value("bl_power"))
+    if mode == "MODE_TINKERBOARD":
+        return not int(_get_value("tinker_mcu_bl"))
```

Comment:
That seems wrong to me. Since `tinker_mcu_bl` apparently only works with brightness, not real power on/off, this function would return `True` for `0` and `False` for `1`-`255`

Reply:
Fixed